easybuild/easyconfigs/m/MultilevelEstimators/MultilevelEstimators-0.1.0-GCC-11.2.0-Julia-1.7.2.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'MultilevelEstimators'
+version = '0.1.0'
+local_julia_ver = '1.7.2'
+versionsuffix = "-Julia-%s" % local_julia_ver
+
+homepage = 'https://github.com/PieterjanRobbe/MultilevelEstimators.jl'
+description = "The Julia module for Multilevel Monte Carlo methods"
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('Julia', local_julia_ver, '-linux-%s' % ARCH, True),
+]
+
+postinstallcmds = [
+    "cd %(installdir)s && mkdir pkgs",
+    """export JULIA_DEPOT_PATH=%%(installdir)s/pkgs &&
+    julia -e 'using Pkg;Pkg.add(url="%s")' """ % homepage
```

Comment:
This basically means that the sources will be downloaded on the fly during the installation, which is definitely something we should try and avoid. What's worse is that the version isn't specified at all, so this will probably just pull in the latest code straight from the repo?